File: <100/006/006.cpp
#include<iostream>
using namespace std;

int sum_linear(int t){
  return t*(t+1)/2;
}

int sum_square(int t){
  return t*(t+1)*(2*t+1)/6;
}

int solve(){
  int s2 = sum_square(100);
  int s  = sum_linear(100);
  return s*s - s2;
}


int main(){
  cout << solve() << endl;
  return 0;
}

File: <100/006/006.jl
square(n:: Int):: Int = n^2
sum_linear(n:: Int):: Int = n * (n+1) ÷ 2
sum_square(n:: Int):: Int = n * (n+1) * (2n+1) ÷ 6

function  main():: Nothing
  sum_squared:: Int = 100 |> sum_linear |> square
  squared_sum:: Int = 100 |> sum_square
  diff:: Int =  sum_squared - squared_sum
  
  diff |> println
  return nothing
end


@time main()
